tests/Agent/IntegrationTests/Shared/Shared.csproj

```diff
@@ -13,7 +13,7 @@
     <PackageReference Include="Common.Logging.Core" Version="3.3.1" />
     <PackageReference Include="CouchbaseNetClient" Version="2.3.8" />
     <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.3" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Not required for landing PR...
Since the version is the same now, can we pull out the PackageReference so that we just have one?